### DISCUSSION TITLE
Moor Autoincrement Error

### DISCUSSION THREAD

**Discussion by DhaiwatMnt** [simolus3/drift#1485] [Q&A]
Using " IntColumn get id => integer().autoIncrement().call(); "

Generates error 

"This getter does not create a valid column that can be parsed by moor."

Earlier it worked fine using call() and without the same.

**Comment by simolus3** [comment]
> Earlier it worked fine using call()

Looking at the generator code, I don't think that `.call()` could have ever worked.

Does your editor change these expressions to `call()` automatically? We can support the `.call()` syntax but I'm wondering how relevant this is.

**Reply by DhaiwatMnt** [reply]
No it doesn't change automatically. But with call() method I don't have to manually add the id, the same cant be said about the default method.
